### ISSUE TITLE
BETA BLOG - Jakarta Mail 2.0 Support

### ISSUE THREAD

**Issue by uberskigeek** [OpenLiberty/open-liberty#15889]
The information you provide here will be included in the Open Liberty beta blog post ([example](https://openliberty.io/blog/2020/08/05/jakarta-grpc-beta-20009.html)), which will be published on [openliberty.io/blog/](https://www.openliberty.io/blog/), and potentially elsewhere, to promote this beta feature/function of Open Liberty.

Please provide the following information the week before the GA/beta date (to allow for review and publishing):

1. Which Liberty feature(s) does your update relate to?
    
   Human-readable name (eg WebSockets feature): JakartaMail
   
   Short feature name (eg websockets-1.0): mail-2.0

2. Who is the target persona? Who do you expect to use the update? eg application developer, operations. Application Developer

3. Write a paragraph to summarises the update, including the following points:
   The Java EE framework has been migrated to the open source Eclipse Jakarta EE Project. As part of this migration JavaMail version 1.6 has been migrated to JakartaMail 2.0. The API package names for the classes previously found under the javax.mail have been migrated to jakarta.mail. 
   The Jakarta mail API as described by the Jakarta Mail FAQ “The Jakarta Mail API is a set of abstract APIs that model a mail system. (Jakarta Mail was previously known as JavaMail.) The API provides a platform independent and protocol independent framework to build Java technology based email client applications. The Jakarta Mail API provides facilities for reading and sending email. Service providers implement particular protocols. Several service providers are included with the Jakarta Mail API package; others are available separately. The Jakarta Mail API is implemented as a Java optional package that can be used on JDK 1.4 and later on any operating system. The Jakarta Mail API is also a required part of the Jakarta EE Platform and the Java Platform, Enterprise Edition (Java EE).”
 Configuring mail sessions works basically the same as with the Liberty Feature JavaMail-1.5 and JavaMail-1.6. They can be configured using the API, or through the server.xml 
 Below is an example of a SMTP Mail session configured through the server.xml:

```
<featureManager>
  <feature>mail-2.0</feature>
  <feature>jndi-1.0</feature>
</featureManager>

<mailSession>
      <mailSessionID>testSMTPMailSession</mailSessionID>
      <jndiName>TestingApp/SMTPMailSessionServlet/testSMTPMailSession</jndiName>
      <description>mailSession for testing SMTP protocol</description>
      <transportProtocol>smtp</transportProtocol>
      <host>localhost</host>
      <user>somuser@someemailserver.com</user>
      <password>usersPassword</password>
      <from>someuser@someemailserver.com</from>
      <property name="mail.smtp.host" value="localhost" \>
      <property name="mail.smtp.port" value="3025" \>
  </mailSession> 
```   
   
   - Where can they find out more about this specific update (eg Open Liberty docs, Javadoc) and/or the wider technology?
Jakarta mail --> https://eclipse-ee4j.github.io/mail/
Administering on liberty substitute JavaMail-1.5 with mail-2.0 —> https://www.ibm.com/support/knowledgecenter/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_admin_javamail.html

## What happens next?
- Add the label for the beta you're targeting: `target:YY00X-beta`.
- Make sure this blog post is linked back to the Epic for this feature/function.
- Your paragraph will be included in the beta blog post. It might be edited for style and consistency.
- You will be asked to review a draft before publication.
- If you would _also_ like to write a standalone blog post about your update (highly recommended), raise an issue on the [Open Liberty blogs repo](https://github.com/OpenLiberty/blogs/issues/new/choose). State in the issue that the blog post relates to a specific release so that we can ensure it is published on an appropriate date (it won't be the same day as the beta blog post).

**Comment by uberskigeek** [comment]
#11860

**Comment by jakub-pomykala** [comment]
Published in 21.0.0.3-beta on 19th of February 2021
